guacamole/src/main/webapp/translations/fr.json

```diff
@@ -130,8 +132,15 @@
 
     },
 
+    "DATA_SOURCE_DEFAULT" : {
+        "NAME" : "Standard (XML)"
+    },
+
     "FORM" : {
 
+        "FIELD_PLACEHOLDER_DATE" : "DD-MM-YYYY",
```

Comment:
Ok got it. I though this was some sort of mask for the date. I just pushed an update.